### PR TITLE
[Demo; do not merge] WASM in JS example

### PR DESCRIPTION
**Pull request by Kevingislason**

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -1,7 +1,3 @@
 nodeLinker: node-modules
 
-plugins:
-  - path: .yarn/plugins/@yarnpkg/plugin-interactive-tools.cjs
```
Comment (Kevingislason) [inline .yarnrc.yml:4]: This is the plugin that was breaking things

File: package.json [hunk 2]
```diff
@@ -25,6 +25,9 @@
         "@polkadot/ui-settings": "^0.69.1",
         "@polkadot/util": "^5.6.3",
         "@polkadot/util-crypto": "^5.6.3",
+        "esbuild": "^0.12.4",
+        "esbuild-wasm": "^0.12.4",
```
Comment (Kevingislason) [inline package.json:29]: These dependencies aren't required for WASM stuff

File: package.json [hunk 3]
```diff
@@ -25,6 +25,9 @@
...
+        "fibonacci": "./fib",
```
Comment (Kevingislason) [inline package.json:30]: Built this from rust using` wasm-pack build` and installed as an npm package

File: src/App.js [hunk 4]
```diff
@@ -1,65 +1,40 @@
-import React, { useState, createRef } from 'react';
-import { BrowserRouter as Router } from 'react-router-dom';
-import { Container, Dimmer, Loader, Grid, Message } from 'semantic-ui-react';
-import 'semantic-ui-css/semantic.min.css';
+import React, { useState } from 'react';
 
-import { SubstrateContextProvider, useSubstrate } from './substrate-lib';
-import { DeveloperConsole } from './substrate-lib/components';
+const Loaded = ({ wasm }) => <button onClick={() => console.log(wasm.fibonacci(3))}>Click me</button>;
```
Comment (Kevingislason) [inline src/App.js:3]: Log fibanocci(3) to console on button press

**Comment by stechu** [comment]
@Kevingislason  seems forget to commit the file `.fib` ?

**Comment by stechu** [comment]
I also get this example working:
https://developer.mozilla.org/en-US/docs/WebAssembly/Rust_to_wasm

**Pull request closed without merging**